### ISSUE TITLE
Is the encoder streamable?

### ISSUE THREAD

**Issue by YoPatapon** [wxt1997/Transformer-Transducer#1]
The encoder of T-T model uses limited left & right context as mentioned in paper. But I noticed that the implementation of the encoder uses the similar mechanism as Transformer-XL but without passing the history memory. How to do streaming encoding and decoding using this code? Thank you very much if you could explain~

**Comment by oshindow** [comment]
Hi, at present, I'm sorry it is still a full attention version. You can add an attention mask to the Encoder during training. Limit the frames to participate in calculating the attention score(Refer to the sequence mask in tt/decoder.py, which only cares about the history info.) Then you can achieve streaming decoding.

**Comment by YoPatapon** [comment]
OK, Thanks ~

**Comment by stefan-falk** [comment]
@okkteam Hi! Is there any progress regarding a streamable T-T model/training?

I have my own Tensorflow-based implementation of the T-T model but I'm struggling making it streamable. 

What do we have to do after having trained a T-T model using a method as described in https://arxiv.org/pdf/2010.11395.pdf?

Thank you for any feedback!
